hare/flows_test.go

```diff
@@ -434,7 +435,8 @@ func Test_multipleCPsAndIterations(t *testing.T) {
 		}
 	}()
 
-	test.WaitForTimedTermination(t, 100*time.Second)
+	// There are 5 rounds per layer and totoalCPs layers and we double for good measure.
+	test.WaitForTimedTermination(t, 2*networkDelay*5*time.Duration(totalCp))
```

Comment:
shouldn't this be longer if we expect one of the round will fail, hence causing the 2nd iteration?

Reply:
Yes I think it should, would we want to double it again then, to account for each consensus process running twice?

Reply:
Actually I see that its just one layer that runs more than a single iteration, so I think we can just add timeout for one more layer.